murageden/converge-level-up
# #155768857 feature(connect recyclerview): connect the recyclerview to the main a…

## What does this PR do?

- Connect the RecyclerView to the main activity of the Java Developers App

## Description of Task to be completed?
- Circle CI & Code Coverage Integration
- Connect RecyclerView layout File to the MainActivity
- Learn About Adapters & AdapterViews

## How should this be manually tested?
- Clone the repository and set up the Android Studio as described on the `README`
- Run the application
- The app should show the screen as shown on the screenshot below

## What are the relevant pivotal tracker stories?
155768857

## Screenshots (if appropriate)?
![image](https://user-images.githubusercontent.com/22506247/43580806-5abe195c-965f-11e8-9602-6120763088c2.png)



## Questions:
N/A
